app/services/platform-apps/index.ts

```diff
@@ -686,16 +692,14 @@ export class PlatformAppsService extends
     });
   }
 
-  toggleEnable(appId: string) {
+  setEnabled(appId: string, enabled: boolean) {
     const app = this.getApp(appId);
     if (app.enabled) {
-      // disabling
       this.appUnload.next(appId);
```

Comment:
You have a bug here.  If I call `setEnabled('foo', true)` and that app is already enabled, we will erroneously send the `appUnload` event.

Reply:
fixing